src/main/webapp/style.css

```
@@ -70,6 +70,16 @@ ul {
 .header-toggle {
   height: 1.5rem;
   width: 1.5rem;
+  transition: 300ms transform ease-in-out;
+}
+
+/*Animate the open/close icon when snapshot is open*/
```

```suggestion
/* Animate the open/close icon when snapshot is open */
```
See https://google.github.io/styleguide/htmlcssguide.html#Protocol for CSS comment examples with surrounding whitespaces.

---

Please also add what kind of animation it is. Animate the [] by [] for example.

---

I've updated the comment to be more specific d8a307d